lighthouse-extension/package.json

```diff
@@ -5,31 +5,11 @@
     "node": ">=8"
   },
   "scripts": {
-    "watch": "gulp watch",
-    "build": "gulp build:production",
-    "debug-build": "node --inspect-brk ./node_modules/.bin/gulp build:production",
-    "test": "mocha test/**/*-test.js"
-  },
-  "devDependencies": {
-    "acorn": "^5.5.3",
-    "babel-plugin-syntax-object-rest-spread": "^6.13.0",
-    "brfs": "^1.6.1",
-    "browserify": "^16.2.0",
-    "del": "^2.2.0",
-    "gulp": "^3.9.1",
-    "gulp-debug": "^2.1.2",
-    "gulp-eslint": "^4.0.2",
-    "gulp-header": "^1.8.9",
-    "gulp-livereload": "^3.8.1",
-    "gulp-tap": "^0.1.3",
-    "gulp-util": "^3.0.7",
-    "gulp-zip": "^3.2.0",
-    "package-json-versionify": "^1.0.4",
-    "run-sequence": "^1.1.5"
-  },
-  "resolutions": {
-    "browserify/insert-module-globals/lexical-scope/astw": "2.2.0",
-    "**/astw/acorn": "5.5.3"
+    "build": "node ./build-extension.js",
+    "test": "mocha test/**/*-test.js",
+    "clean": "rimraf dist",
+    "package": "node ./build-extension.js package"
   },
+  "devDependencies": {},
```

Comment:
they were all devDeps (so no npm package penalty), we no longer support an environment requiring only one version of any npm module, there were even fewer deps now, and I got a few typescript conflicts between here and the parent directory, so it made sense to move them all up to the root directory